at_boundaries: {
  room_1: [[0.5103914737701416, -0.49582797288894653], [-1.862577199935913,-0.432850182056427], [-1.9274754524230957, 0.6790512204170227], [0.465421587228775,0.7207310795783997]],
  room_2: [[0.4853273928165436, -1.550743818283081], [-1.7459230422973633,-1.6497180461883545], [-1.8296730518341064,-0.7239020466804504], [0.508831262588501,-0.7111300230026245]]
}

